### DISCUSSION TITLE
kubernetes_logs source in GCP doesn't work

### DISCUSSION THREAD

**Discussion by madhur-df** [vectordotdev/vector#20865] [Q&A]
They do not work at all and there is no flow of logs.

I am using Helm Chart for vector agents as daemon-sets.

The directory on the node `/var/log/pods` is already provided by `hostPath` by the Helm Chart.

![Screenshot 2024-07-09 at 8 23 18 PM](https://github.com/user-attachments/assets/4da5a22a-cfd2-44be-a7e2-fd789c244313)

The same setup works on AWS.
Is there anything to be done on GCP side for access or something?

**Comment by madhur-df** [comment]
Can someone please help here?

**Reply by jszwedko** [reply]
I'm not familiar with k8s on GCP. You could try cross-posting this into the [Discord server](https://chat.vector.dev) to see if you get more of a response there.

**Reply by madhur-df** [reply]
@jszwedko it is the same issue as https://github.com/vectordotdev/vector/issues/18072

The only difference is my K8s cluster runs on GKE / GCP
